dashboard/test/ui/features/star_labs/musiclab/musiclab_drag_block.feature

```diff
@@ -27,5 +27,5 @@ Scenario Outline: Dragging play sound block
 
 Examples:
   | url                                                       | test_name               |
-  | http://studio.code.org/s/allthethings/lessons/46/levels/2 | music lab script level  |
+  | http://studio.code.org/s/allthethings/lessons/46/levels/4 | music lab script level  |
```

Comment:
Should we change this to a standalone project URL (projects/music/new) so we don't need to worry about updating it if the allthethings progression changes?

Reply:
We depend on `allthethings` for much of our UI testing, so I think it's reasonable to keep these particular levels in sync with UI tests.